packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -821,7 +821,8 @@ export default createRule<Options, MessageId>({
       | 'string'
       | 'truthy boolean'
       | 'truthy number'
-      | 'truthy string';
+      | 'truthy string'
+      | 'branded boolean';
```

Comment:
[Style] Super nitpicky, but - these _were_ in alphabetical order. Up for alphabetizing?